public/js/toggle-label.js

```diff
@@ -0,0 +1,13 @@
+let checked = false;
+
+function toggleLabel() {
+  checked = !checked;
+  const nodes = View.nodes;
+  Object.values(nodes).forEach((node) => {
```

Comment:
have to use ``Object.keys`` here to get the nodes reference name as opposed to the display name! Work perfectly otherwise, going to add a commit for this then merge.